final/design/eval-strat.tex

```diff
@@ -0,0 +1,38 @@
+\section{Evaluation Strategies}
+\label{sec:eval-strat}
+As explained in \cref{sec:supp-diff-ways}, there are multiple ways for
+specifying the dynamic semantics of a language. Interpretation can for example
+be performed through an interpreter generated from a DynSem specification, or by
+implementing the interpreter in Stratego or Java. However, in the end all of
+these methods do the same thing, that is they accept an AST as input and somehow
+transform it into an output. In Spoofax, both the input and the output can be
+represented as a Stratego term. It is only the internal representation of the
+evaluation context and the implementation of the interpreter that is different.
+
+The Strategy pattern is useful for computation task that have the same types of
```

Comment:
Previously design patterns are all lower case, so I suggest doing that here as well.